_arduino/LiDARMinimal/LiDARMinimal.ino

```diff
@@ -0,0 +1,117 @@
+#include <Wire.h>
```

Comment:
Please:

- use a formatter on this file
- group global variables together by use
- put comments above the line instead of at the end